microhapulator/cli/pipe.py

```diff
@@ -29,10 +31,10 @@ def main(args):
     config = dict(
         samples=samples,
         readfiles=workingfiles,
-        mhrefr=Path(args.markerrefr).resolve(),
-        mhdefn=Path(args.markerdefn).resolve(),
-        hg38path=Path(args.hg38).resolve(),
-        hg38index=Path(args.hg38idx).resolve(),
+        mhrefr=str(Path(args.markerrefr).resolve()),
+        mhdefn=str(Path(args.markerdefn).resolve()),
+        hg38path=str(Path(args.hg38).resolve()),
+        hg38index=str(Path(args.hg38idx).resolve()),
```

Comment:
Passing paths here also caused an issue with newer versions of Snakemake where they didn't with version 7. 🤷‍♂️